openedx/core/djangoapps/oauth_dispatch/dot_overrides.py

```diff
@@ -72,7 +74,7 @@ def save_bearer_token(self, token, request, *args, **kwargs):
         """
         grant_type = request.grant_type
         user = request.user
-
+        log.info('Starting refresh token and access token revocation and creation process for user[%d].', user)
```

Comment:
The log statement should be aligned with the purpose of the method. Also, if you suspect duplicates, log a portion of the token to de-dupe.

```python
_token_prefix = token['access_token'][:3]
log.info('Saving bearer token with prefix [%s] for user [%d]', _token_prefix, user.id)
```